src/ServiceManager/Travely.ServiceManager.Service/Services/ActivityService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Grpc.Core;
+using System.Threading.Tasks;
+
+namespace Travely.ServiceManager.Service
+{
+    public class ActivityService : ActivityProto.ActivityProtoBase
+    {
+        private readonly IActivityManager _activityManager;
+
+        public ActivityService(IActivityManager activityManager)
+        {
+            _activityManager = activityManager;
+        }
+        public override async Task<ActivityResponce> CreateActivity(Activity activity, ServerCallContext context)
+        {
+            var activityId = await _activityManager.CreateActivity(activity);
+
+            return new ActivityResponce
+            {
+                Message = $"Successfully cerated new activity: {activity.Name}",
```

Comment:
```suggestion
                Message = $"Successfully created new activity: {activity.Name}",
```

Reply:
Better to get activity.Name from ActivityManager, ActivityService can't be sure what has been added when CreateActivity() method called.

Reply:
done